src/chatgpt-api.ts

```diff
@@ -16,6 +16,8 @@ export class ChatGPTAPI {
   protected _apiBaseUrl: string
   protected _backendApiBaseUrl: string
   protected _userAgent: string
+
+  // stores access tokens for up to 10 seconds before needing to refresh
   protected _accessTokenCache = new ExpiryMap<string, string>(10 * 1000)
```

Comment:
Is there a specific reason for a 10 second refresh? Could we potentially go longer?

Reply:
Good call; yep, we could definitely go longer. I can make this configurable via the constructor.

I'm honestly not sure of a good default value. If you want to do some testing to figure out what the web UI is doing, that'd be really helpful.